.github/CHANGELOG.md

```diff
@@ -12,6 +12,8 @@
 
 * Adds the function `hafnian_banded` to calculate the hafnian of a banded matrix [#246](https://github.com/XanaduAI/thewalrus/pull/246)
 
+* Adds the functions `hermite_multidimensional_numba` and `grad_hermite_multidimensional_numba` to calculate the multidimensional Hermite polynomials and its gradients using numba [#251](https://github.com/XanaduAI/thewalrus/pull/251)
```

Comment:
@nquesada are we going to call them `hermite_multidimensional` because they are not the regularized ones? or are they regularized?

Reply:
They correspond to the `renorm = True` version of `hermite_polynomials`